docassemble/ServiceByAltMeansAndDiligentSearch/data/questions/about_the_case.yml

```
@@ -110,7 +110,7 @@ if address_unknown:
 ---
 id: last lived at address
 question: |
-  When did ${ other_parties[0].name } last live at the above address?
+  When was the last time ${ other_parties[0].name } lived at the address you mentioned?
```

We can show this address, right? Something like this:

```suggestion
  When was the last time ${ other_parties[0].name } lived at the address you mentioned (${ other_parties[0].address.address})?
```